package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "gutenberg",
-	"version": "5.3.0",
+	"version": "5.4.0",
```

Comment:
It seems like your branch isn't aligned with the latest `master` changes 🤷‍♂️

Reply:
Yeah, I still need to rebase. I was having some trouble rebasing so am figuring that out

Reply:
Just rebased! The version should be updated now